Anlab.Core/Domain/TestItem.cs

```diff
@@ -29,6 +29,7 @@ public class TestItem
         public bool Public { get; set; }
 
         public string Notes { get; set; }
+        public string NotesEncoded { get; set; }
```

Comment:
@laholstege What do you think about not having NotesEncoded be a settable property and just have it only have a getter.  That getter would be implemented to automatically encode the notes property, that way you wouldn't have to worry about setting it in the db initializer and every other place you need to use it